modules/ossm-install-rosa.adoc

```diff
@@ -0,0 +1,84 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2/ossm-create-smcp.adoc
+////
+
+:_content-type: REFERENCE
+[id="ossm-install-rosa_{context}"]
+= Installing on Red Hat OpenShift Service on AWS (ROSA)
+
+Starting with version 2.2, {SMProductName} supports installation on Red Hat OpenShift Service on AWS (ROSA). This section documents the additional requirements when installing Service Mesh on this platform.
+
+== Installation location
```

Comment:
Please add in anchor IDs for all new headings per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings